ext/hivemq-edge-openapi-2024.8-proposal.yaml

```diff
@@ -0,0 +1,5561 @@
+            created.
+          readOnly: true
+        id:
+          type: string
+          description: The unique identifier of the policy.
+        lastUpdatedAt:
+          type: string
+          format: date-time
+          description: The formatted UTC timestamp indicating when the policy was
+            updated the last time.
+          readOnly: true
+        matching:
+          $ref: '#/components/schemas/DataPolicyMatching'
+        onFailure:
+          $ref: '#/components/schemas/DataPolicyAction'
+        onSuccess:
+          $ref: '#/components/schemas/DataPolicyAction'
+        validation:
+          $ref: '#/components/schemas/DataPolicyValidation'
+      required:
+        - id
+        - matching
+    DataPolicyAction:
+      type: object
+      description: "One or more operations the outcome of the validation triggers.\
+        \  When this field is empty, the outcome of the policy validation does not\
+        \ trigger any operations."
+      properties:
+        pipeline:
+          type: array
+          description: "The pipeline to execute, when this action is triggered. The\
+            \ operations in the pipeline are executed in-order."
+          items:
+            $ref: '#/components/schemas/PolicyOperation'
+    DataPolicyList:
+      type: object
+      description: A listing of data policies.
+      properties:
+        _links:
+          $ref: '#/components/schemas/PaginationCursor'
+        items:
+          type: array
+          description: List of result items that are returned by this endpoint
+          items:
+            $ref: '#/components/schemas/DataPolicy'
+    DataPolicyMatching:
+      type: object
+      description: The matching rules the policy applies.
+      properties:
+        topicFilter:
+          type: string
+          description: The topic filter for which the policy is matched.
+      required:
+        - topicFilter
+    DataPolicyValidation:
+      type: object
+      description: "The section of the policy that defines how incoming MQTT messages\
+        \ are validated. If this section is empty, the result of the policy validation\
+        \ is always successful."
+      properties:
+        validators:
+          type: array
+          description: The validators of the policy.
+          items:
+            $ref: '#/components/schemas/DataPolicyValidator'
+    DataPolicyValidator:
+      type: object
+      description: A policy validator which executes the defined validation.
+      properties:
+        arguments:
+          type: object
+          description: The required arguments of the referenced validator type.
+        type:
+          type: string
+          description: The type of the validator.
+          enum:
+            - SCHEMA
+      required:
+        - arguments
+        - type
+    DeviceDataPoint:
+      ### TODO We need to name that entity
+      type: object
+      description: A data point on a device connected to the adapter
```

Comment:
properties missing

Reply:
I've added additionalProperties: true to the spec

Reply:
Also renamed it `DomainTagAddress` for the time being